src/main/kotlin/pl/edu/agh/server/config/JwtService.kt

```diff
@@ -30,29 +38,49 @@ class JwtService {
     }
 
     fun isTokenValid(token: String, userDetails: UserDetails): Boolean {
```

Comment:
Ta funkcja wyglada git. Jak pozniej bedziemy z tego korzystac w kontrolerach? Bedzie trzeba wolac te funkcje za kazdym razem? Jak tak to moze dorzucimy ja do filtru zeby dziala juz wszedzie?

Reply:
już jest w filtrze